From 3b556f11e3d4ea8d4fe65aa7ea47fc8aa3558b45 Mon Sep 17 00:00:00 2001
From: "Vadim V. Vlasov" <vadim.vlasov@t-platforms.ru>
Date: Fri, 17 May 2019 16:19:49 +0300
Subject: [PATCH 2/6] Change shell name in script headers

Signed-off-by: Vadim V. Vlasov <vadim.vlasov@t-platforms.ru>
---
 getrom.sh  | 2 +-
 netconf.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/getrom.sh b/getrom.sh
index faf7616..13ae1f0 100644
--- a/getrom.sh
+++ b/getrom.sh
@@ -1,4 +1,4 @@
-#!/bin/ash
+#!/bin/sh
 
 VAR_PREFIX=/var/run/recoveryui/
 URL=$1
diff --git a/netconf.sh b/netconf.sh
index 57dd7af..73c0346 100644
--- a/netconf.sh
+++ b/netconf.sh
@@ -1,4 +1,4 @@
-#!/bin/ash
+#!/bin/sh
 
 IP=$1
 NM=$2
-- 
2.7.4

