From cd005381356cbf91778d056f8fd37a585da4941d Mon Sep 17 00:00:00 2001
From: "Vadim V. Vlasov" <vadim.vlasov@t-platforms.ru>
Date: Fri, 17 May 2019 16:56:43 +0300
Subject: [PATCH 5/6] Make sure that rfs_version and kernel_release strings are
 null terminated

Signed-off-by: Vadim V. Vlasov <vadim.vlasov@t-platforms.ru>
---
 src/main_page.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main_page.c b/src/main_page.c
index 8b3cc58..b6b4a4d 100644
--- a/src/main_page.c
+++ b/src/main_page.c
@@ -219,8 +219,8 @@ static struct {
   char voltage_label[20];
   char mfg_date_label[20];
   char bmc_version_label[20];
-  char rfs_version[20];
-  char kernel_release[20];
+  char rfs_version[21];
+  char kernel_release[21];
   char year_val[5];
   char month_val[3];
   char day_val[3];
-- 
2.7.4

